multimedia/mat2/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                mat2
+version             0.13.3
+categories          multimedia
```

Comment:
this should probably become `categories-prepend` ; in that way the "python" category set by the PG will still be present as well